.github/workflows/build-develop.yml

```diff
@@ -330,7 +330,7 @@ jobs:
           pip3 install --upgrade pip
           pip3 install -r requirements.txt
           pip3 install -r requirements.txt
-          behavex -t=@smoke
+          behavex -t=@smoke --parallel-processes=4 --parallel-scheme=scenario
```

Comment:
why only 4 processes? i guess we can use more.. i'm running locally with 20.. but you can put at least 10.. to make it faster..

Reply:
Ok great. So I will change :)